package proto_ff;

import "yd_fieldoptions.proto";


message E_SensitivewordSensitive_word
{
	optional int32 m_id = 1[(yd_fieldoptions.field_cname) = "序号"];
	optional string m_sensitiveword = 2[(yd_fieldoptions.field_cname) = "敏感字", (yd_fieldoptions.field_bufsize) = 60];
}


message Sheet_SensitivewordSensitive_word
{
	repeated E_SensitivewordSensitive_word E_SensitivewordSensitive_word_List = 1[(yd_fieldoptions.field_arysize)=6000];
}

message E_SensitivewordRolesensitive
{
	optional int32 m_id = 1[(yd_fieldoptions.field_cname) = "序号"];
	optional string m_sensitiveword = 2[(yd_fieldoptions.field_cname) = "敏感字", (yd_fieldoptions.field_bufsize) = 60];
}


message Sheet_SensitivewordRolesensitive
{
	repeated E_SensitivewordRolesensitive E_SensitivewordRolesensitive_List = 1[(yd_fieldoptions.field_arysize)=300];
}
